src/app-layer.c

```diff
@@ -99,6 +101,18 @@ static void AppLayerIncFlowCounter(uint8_t ipproto, AppProto alproto, ThreadVars
     }
 }
 
+void AppLayerIncTxCounter(uint8_t ipproto, AppProto alproto, ThreadVars *tv,
```

Comment:
I think every function that has Threadvars as an argument puts it first in the arg list. Might be good to do that here as well.